libs/sdk/skyux-eslint/src/rules/template/prefer-label-text.ts

```diff
@@ -75,6 +75,16 @@ function getAttributeByName(
   return;
 }
 
+/**
+ * Checks if the provided element contains Angular control flow syntax (@if, @for, @switch, etc.).
+ */
+function hasControlFlowSyntax(el: TmplAstElement): boolean {
+  const sourceText = el.sourceSpan.toString();
+  const controlFlowPatterns =
+    /@(if|for|switch|case|default|else|defer|placeholder|loading|error)\b/;
```

Comment:
Since `@` is now a reserved character in templates, rather than try to maintain a complete list, can it just look for a `@` followed by characters?

Reply:
Good feedback!